go-controller/pkg/cluster/gateway_init.go

```diff
@@ -11,33 +11,37 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 )
 
-// bridgedGatewayNodeSetup makes the bridge's MAC address permanent, sets up
+// bridgedGatewayNodeSetup makes the bridge's MAC address permanent (if needed), sets up
 // the physical network name mappings for the bridge, and returns an ifaceID
 // created from the bridge name and the node name
-func bridgedGatewayNodeSetup(nodeName, bridgeInterface string) (string, string, error) {
+func bridgedGatewayNodeSetup(nodeName, bridgeName, bridgeInterface string, brCreated bool) (string, string, error) {
 	// A OVS bridge's mac address can change when ports are added to it.
 	// We cannot let that happen, so make the bridge mac address permanent.
 	macAddress, err := util.GetOVSPortMACAddress(bridgeInterface)
 	if err != nil {
 		return "", "", err
 	}
-	stdout, stderr, err := util.RunOVSVsctl("set", "bridge",
-		bridgeInterface, "other-config:hwaddr="+macAddress)
-	if err != nil {
-		return "", "", fmt.Errorf("Failed to set bridge, stdout: %q, stderr: %q, "+
-			"error: %v", stdout, stderr, err)
+	if brCreated {
+		var err error
```

Comment:
This error is not used; can be removed.

Reply:
@dcbw If remove the error at L25, I am getting lint error.